src/EFCoreSecondLevelCacheInterceptor/EFInfinispanCacheServiceProvider.cs

```diff
@@ -21,17 +21,17 @@ public EFInfinispanCacheServiceProvider()
     }
     public void ClearAllCachedEntries()
     {
-        throw new NotImplementedException();
+        cache.Clear().Wait();
     }
 
     public EFCachedData GetValue(EFCacheKey cacheKey, EFCachePolicy cachePolicy)
     {
-        throw new NotImplementedException();
+        return cache.Get(cacheKey).Result;
     }
 
     public void InsertValue(EFCacheKey cacheKey, EFCachedData value, EFCachePolicy cachePolicy)
```

Comment:
Since `EFCachePolicy` is not included yet, another issue for the same can be created right?

Reply:
sure